lib/process_settings/testing/helpers.rb

```diff
@@ -9,9 +9,6 @@
 module ProcessSettings
   module Testing
     module Helpers
-      extend self
```

Comment:
I confirmed that `extend self` wasn't useful when the code was run by callers--it was just used by the test of this code.